src/cli/migrate/nodeVersion/index.ts

```diff
@@ -14,12 +14,45 @@ type SubPatch = (
   replace: string;
 };
 
+type VersionResult = {
+  version: string;
+  err: string | undefined;
+};
+
+type RegistryResponse = {
+  'dist-tags': Record<string, string>;
+};
+
+export const getLatestNodeTypes = async (): Promise<VersionResult> => {
```

Comment:
Hmmm we might not want the latest latest, because Node releases odd number versions as unstable so there may be Node 23 types coming soon. If possible we want to get the latest version 22 types